custom/enikshay/integrations/bets/repeaters.py

```diff
@@ -242,3 +250,52 @@ def create_case_repeat_records(sender, case, **kwargs):
     create_repeat_records(BETSAYUSHReferralRepeater, case)
 
 case_post_save.connect(create_case_repeat_records, CommCareCaseSQL)
+
+
+class BETSUserRepeater(Repeater):
```

Comment:
Yeah, I was thinking about that too - I wanted to start them this way so I wouldn't have a mental block from doing something BETS-specific if it would've been the smarter approach.  After having finished writing them, they are still generic, so maybe I will genericize them.